tests/test_utils.py

```diff
@@ -346,3 +347,61 @@ def test_get_compile_command_sort_args(tmpdir_cwd):
             "--no-annotate --no-emit-index-url --no-emit-trusted-host "
             "requirements.in setup.py"
         )
+
+
+@pytest.mark.parametrize(
+    ("values", "key"),
+    (
+        pytest.param(
+            ("foo", "bar", "baz", "qux", "quux"),
+            lambda s: s[0],
```

Comment:
I think it's more elegant to do
```suggestion
            operator.itemgetter(0),
```

Reply:
Yeah, `operator` looks better. Thanks! Addressed in 080f85.